src/tests/unit/ingredients-dao.test.js

```diff
@@ -0,0 +1,145 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import config from 'config';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+import { getIngredientsData } from 'tests/unit/test-data';
+import { openapi } from 'utils/load-openapi';
+import { GetFilterProcessor } from 'utils/process-get-filters';
+
+const ingredientsGetParameters = openapi.paths['/ingredients'].get.parameters;
+
+sinon.restore();
+
+sinon.replace(config, 'get', () => ({ oracledb: {} }));
+
+chai.should();
+chai.use(chaiAsPromised);
+chai.use(sinonChai);
+
+describe('test ingredients dao', () => {
+  const {
+    getIngredientsQuery,
+    emptyGetIngredientsQuery,
+    testFilters,
+    ingredientAliases,
+    testBindParams,
+    testSerializerReturn,
+    testConditionals,
+    testConnectionReturn,
+    testConnectionReturnRows,
+  } = getIngredientsData;
+  let serializeIngredientsStub;
+  let getConnectionStub;
+  let connectionSpy;
+  let ingredientsDao;
+  let getFilterProcessorStub;
+  let getFilterProcessorContructorStub;
+  beforeEach(() => {
+    connectionSpy = sinon.stub().returns(testConnectionReturn);
+    serializeIngredientsStub = sinon.stub().returns(testSerializerReturn);
+    getConnectionStub = sinon.stub().resolves({
+      execute: connectionSpy,
+      close: () => {},
+    });
+    getFilterProcessorContructorStub = sinon.stub();
+    getFilterProcessorStub = sinon.stub(GetFilterProcessor.prototype, 'processGetFilters').returns({
+      bindParams: testBindParams,
+      conditionals: testConditionals,
+    });
+
+    Object.setPrototypeOf(GetFilterProcessor, getFilterProcessorContructorStub);
+
+    ingredientsDao = proxyquire('../../api/v1/db/oracledb/ingredients-dao', {
```

Comment:
For this first path passed to `proxyquire`, you can just use `'api/v1/...'` since you want the path to evaluate relative to the current module. It's just the path keys in the object you pass that need to be relative to the proxyquired module.